ci/tasks/prepare-container-bosh-docker-build-context.yml

```diff
@@ -5,7 +5,6 @@ image_resource:
   type: docker-image
   source:
     repository: ubuntu
-    tag: "14.04"
```

Comment:
This will always pull the latest version of `ubuntu`, `noble` atm. Is that ok?

Reply:
yes, the ubuntu docker image used in other task use latest: https://github.com/cloudfoundry/bosh/blob/main/ci/pipeline.yml#L1427

The bash script this file runs only uses `cp` and `mkdir`

Reply:
also the `14.04` line is `trusty` which is older than the now out of support `xenial`